Updated Material container transition

atsansone opened the pull request:
Replaces the transition animations with a zoom-in, zoom-out animation.

Fixes #8759
Fixes #8504

Staged: https://tony-flutter-site--fix-8759-5avd4gju.web.app/platform-integration/platform-adaptations#navigation-transitions

HansMuller reviewed (approved):
Nice animation! LGTM

sfshaza2 commented on src/platform-integration/platform-adaptations.md:
```diff
@@ -89,14 +89,7 @@ On **iOS**:
-  [slides up and fades in][].
-* On API 28 and later, the bottom-up animation
-  [slides and clip-reveals up][].
+On **Android**, Flutter uses the [`ZoomPageTransitionsBuilder`][] animation.
```
What does that transition do? Can you explain very briefly, like the original text?

atsansone replied:
Done.

sfshaza2 reviewed (changes requested):
Please rename this page to match the sidebar.

I wish you would put a very brief description of what this PR does when you create the PR. Having to read open issues is frustrating. For example, I'm not sure why you added/modified all that HTML. I don't see tabs in the generated page. What is it for?

I know that Xiao added all those GIFs and that you didn't do it, but geez, it makes me jumpy and nervous.

atsansone commented:
@sfshaza2 : Updated the PR summary, changed the title, updated the animation description.

The HTML just matches the existing structures for the page UX (table plus image with captions). Nothing new was done.

sfshaza2 reviewed (approved):
lgtm
